zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -84,6 +87,20 @@ public ZkClientUriDomainMappingHelper(ZooKeeperServer zks) {
     parseZNodeMapping();
   }
 
+  /**
+   * @return True if the new updater is setup to the helper instance. False if the specified updater is not set since
+   * another updater has already been configured.
+   */
+  synchronized boolean setDomainAuthUpdater(ConnectionAuthInfoUpdater updater) {
+    if (this.updater != null) {
+      LOG.error("Client connection ACL updater has been setup. Skip setting up new updater.");
+      return false;
+    } else {
```

Comment:
Most parts of ZK codebase use Guard Clause unless there are additional if conditions to check for. That way it's less verbose and clear. So it would be great if we could remove the redundant `else`. The [contributor guideline](https://cwiki.apache.org/confluence/display/ZOOKEEPER/HowToContribute) asks contributors to follow the style of the existing code.

Reply:
Sure, I don't have a strong preference here. I will just follow the style that we are using.